--- a/src/applications/pc_boxes/box_app_manager.c
+++ b/src/applications/pc_boxes/box_app_manager.c
@@ -3993,7 +3993,7 @@
     for (int i = 0; i < selection->selectedMonCount; i++) {
         posInBox = selectionTopLeftPos + (selection->selectedMonsOrigBoxPos[i] - origSelectionTopLeftPos);
         PCBoxes_TryStoreBoxMonAt(boxAppMan->pcBoxes, USE_CURRENT_BOX, posInBox, boxMon);
-        (u8 *)boxMon += boxMonStructSize;
+        boxMon = (BoxPokemon *)((u8 *)boxMon + boxMonStructSize);
     }
 
     BoxApp_SetPreviewedBoxMon(boxApp, PCBoxes_GetBoxMonAt(boxAppMan->pcBoxes, USE_CURRENT_BOX, BoxApp_GetCursorBoxPosition(boxApp)));
