--- a/arm9/overlays/11/asm/ov11_02242B78.s
+++ b/arm9/overlays/11/asm/ov11_02242B78.s
@@ -7452,7 +7452,7 @@
 	bne _022464E0
 	ldr r1, [sp, #8]
 	add r0, r5, #0
-	bl ov11_0224C6D4
+	bl PcDp_MoveIsOnDamagingTurn
 	cmp r0, #0
 	beq _022464E0
 	ldr r0, [sp, #0x54]
@@ -7700,7 +7700,7 @@
 	bne _022466B6
 	ldr r1, [sp]
 	add r0, r7, #0
-	bl ov11_0224C6D4
+	bl PcDp_MoveIsOnDamagingTurn
 	cmp r0, #0
 	beq _022466B6
 	ldr r1, [r6]
